angrmanagement/__main__.py

```diff
@@ -90,7 +90,7 @@ def drawContents(self, painter):
             info["CFBundleName"] = name
         except Exception as e:  # pylint: disable=broad-except
             # This happens before logging is setup so use stderr
-            print(f"{type(e).__name__}: {e}", file=sys.stderr)
+            print(f"Failed to set App name! {type(e).__name__}: {e}", file=sys.stderr)
```

Comment:
Wasn't this a different PR?

Reply:
It's just cleaning up the error message. I had to do it for debugging something earlier, left it in because it's a good change that'll save someone else debugging later should this ever happen to them for some reason.